packages/request-manager/src/torIdentities.ts

```diff
@@ -0,0 +1,91 @@
+import http from 'http';
+import https from 'https';
+import { SocksProxyAgent } from 'socks-proxy-agent';
+import { EventEmitter } from 'events';
+import { RequestInit } from 'node-fetch';
+
+interface TorConnectionOptions {
+    host: string;
+    port: number;
+}
+
+interface TorRequestOptions {
+    identity?: string;
+}
+
+export class TorIdentities extends EventEmitter {
```

Comment:
is extension of EventEmitter necessary?

Reply:
Not necessary, removed 9d574d83e0e1889a8afa6b20f42fcac7de1a21be